### PR TITLE
Refactor supported countries

### PR DESCRIPTION
**Pull request by istanishev**

### CODE REVIEW

File: src/PostalCodes/IsoCountryCodeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace PostalCodes
+{
+    /// <summary>
+    /// Class IsoToVistaprintCountryCodeConverter.
+    /// </summary>
+    public class IsoCountryCodeConverter
+    {
+        /// <summary>
+        /// Gets the vistaprint country code.
```
Comment (brendanmshea) [inline src/PostalCodes/IsoCountryCodeConverter.cs]: Should change the comment to remove 'vistaprint'.

File: src/PostalCodes/IsoCountryCodeConverter.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+
+namespace PostalCodes
+{
+    /// <summary>
+    /// Class IsoToVistaprintCountryCodeConverter.
```
Comment (brendanmshea) [inline src/PostalCodes/IsoCountryCodeConverter.cs]: Fix comment (though this comment really doesn't add anything, so perhaps we should just remove it.)

File: src/PostalCodes.UnitTests/IsoCountryCodeConverterTests.cs [hunk 3]
```diff
@@ -15,7 +15,7 @@ internal class IsoToVistaprintCountryCodeConverterTests
         public void GetVistaprintCountryCode_VPSpecificCountryCode_ReturnsVistaprintCountryCode(string input,
```
Comment (brendanmshea) [inline src/PostalCodes.UnitTests/IsoCountryCodeConverterTests.cs]: Fix test name (method name, 'VPSpecific', ReturnsVistaprint).

File: src/PostalCodes.UnitTests/IsoCountryCodeConverterTests.cs [hunk 4]
```diff
@@ -25,7 +25,7 @@ internal class IsoToVistaprintCountryCodeConverterTests
         [TestCase("DE")]
         public void GetVistaprintCountryCode_NonVPSpecificCountryCode_ReturnsSameCountryCode(string input)
```
Comment (brendanmshea) [inline src/PostalCodes.UnitTests/IsoCountryCodeConverterTests.cs]: Fix test name (method name, 'NonVPSpecific').

File: src/PostalCodes/IsoCountryCodeConverter.cs (hunk 2 above)
Comment (rnowosielski) [inline src/PostalCodes/IsoCountryCodeConverter.cs]: All public elements need a documentation, otherwise it will generate error.

Comment (brendanmshea) [inline src/PostalCodes/IsoCountryCodeConverter.cs]: Ok.  Let's make it better than just the class name, then. :)

**Comment by brendanmshea** [comment]
Cool.  where's the approve button? :+1:

**Comment by rnowosielski** [comment]
I think when you approve, you have to merge it. but :+1:  works too!

**Comment by istanishev** [comment]
Any idea why the build is "queued"? I have no idea if it will be built as I was playing with Xamarin and neither nuget nor sandcastle can be used directly/easily (I haven't read if it's possible at all... maybe tomorrow ;)).

**Comment by istanishev** [comment]
Hehe, it just changed to "All is well" ;)

**Comment by rnowosielski** [comment]
Yeah, those CI servers that are free are quite slow: so we shouldn't be pushing every commit to PR.

**Pull request merged**